apps/service_receive/lib/receive/acception/store.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Receive.Acception.Store do
+  @instance Receive.Application.instance()
+  @collection "acceptions"
```

Comment:
This... hurts to read. It just, is not a word that should ever exist.

Reply:
`accepting`? `acceptables`?

Reply:
changed to `accepts`